app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/cohort/CohortPixelInterceptor.kt

```diff
@@ -72,6 +72,7 @@ class CohortPixelInterceptor @Inject constructor(
         private const val PIXEL_PREFIX = "m_atp_"
         private val EXCEPTIONS = listOf(
             "m_atp_ev_enabled_onboarding_",
+            "m_atp_unprotected_apps_bucket_"
```

Comment:
Added this on the exception list as we don't want the pixel to contain any cohort info.